CesiumGltf/generated/include/CesiumGltf/Class.h

```diff
@@ -34,5 +34,21 @@ struct CESIUMGLTF_API Class final : public CesiumUtility::ExtensibleObject {
    * matching the regular expression `^[a-zA-Z_][a-zA-Z0-9_]*$`.
    */
   std::unordered_map<std::string, CesiumGltf::ClassProperty> properties;
+
+  int64_t getSizeBytes() const {
+    int64_t accum = 0;
+    accum += sizeof(Class);
+    if (this->name) {
+      accum += this->name->size();
+    }
+    if (this->description) {
+      accum += this->description->size();
+    }
+    for (auto& [k, v] : this->properties) {
+      accum += k.size();
+      accum += v.getSizeBytes() - sizeof(CesiumGltf::ClassProperty);
+    }
```

Comment:
`std::unordered_map`'s API doesn't seem to give us a clear way to understand its size. But something like this should be pretty good, and definitely close enough for our purposes:

```suggestion
    accum += this->properties.bucket_count() * (sizeof(std::string) + sizeof(CesiumGltf::ClassProperty));
    for (const auto& [k, v] : this->properties) {
      accum += k.capacity() * sizeof(char) - sizeof(std::string);
      accum += v.getSizeBytes() - sizeof(CesiumGltf::ClassProperty);
    }
```